### DISCUSSION TITLE
the voxel size and rotation matrix of output images are different from those of input images.

### DISCUSSION THREAD

**Discussion by ruitians** [jcreinhold/intensity-normalization#85] [Q&A]
I tried to normalize the bunch  of nii.gz files in a folder called t2w with a white matter masks under folder wmmaks. I applied:
nyul-normalize t2w   -o t2_norm -mo t2  -p
I got normalized images under folder t2_norm.  However, the rotation matrix is diagonal matrix and the voxel size is 1x1x1 (the input is 0.7x0.7x0.7).  I also saw negative intensity in the output images (They are all positive in the input images).

**Comment by EBirnie** [comment]
I have the same problem using the nyul-normalize function. The left/right orientation of the normalized images is mirrored. In addition, the origin is no longer defined in the output images.

**Comment by So-cean** [comment]
also meet negative intensity in normalizing t1w images using ```NyulNormalizer```
